server/package.json

```diff
@@ -4,8 +4,8 @@
 	"description": "",
 	"main": "index.ts",
 	"scripts": {
-		"start": "NODE_ENV=production node dist/index.js",
-		"dev": "NODE_ENV=development nodemon index.ts",
+		"start": "cross-env NODE_ENV=production node dist/index.js",
```

Comment:
what are your thoughts on enforcing use of WSL on windows? I'd rather not have to deal with windows-specific anything: environment, file-systems, etc.

Reply:
I don't foresee any other inconveniences for Windows users.
Enforcing WSL might turn down potential contributors.

But I don't have a strong opinion 😅.
I am currently using Windows, I haven't setup WSL yet, and wanted to contribute; therefore I **had** to make this change locally.

If we do want to go that route, let me know and I'll open a PR in the README.md 😸